portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,39 +25,47 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
   
-  private ArrayList<String> quotes;
-
-  public void init() {
-    quotes = new ArrayList<>();
-    quotes.add("Welcome");
-    quotes.add("Bienvenidos");
-    quotes.add("Bonjour");
-    quotes.add("Aloha");
-  }
+  private List<String> entries = new ArrayList<String>();
```

Comment:
This can be private final, since you don't need to reassign the entries variable.

Reply:
Done